docs/es/README.md

```diff
@@ -1,22 +1,16 @@
-<!--
-parent:
-  order: false
-layout: home
--->
-
```

Comment:
@dcwalk did you confirm this doesen't break the vue build? i had thought these comments were necessary for some part of the build process

Reply:
Confirmed just now by building, a index is generated in with the page contents:
![Screenshot_2021-12-09_09-40-32](https://user-images.githubusercontent.com/4692834/145417436-bb78a067-e675-4d3d-8f52-c32aa1a69962.png) 

I removed frontmatter here to match zh and ko READMEs, but happy to add to all if preference is for that. 

If you are referring to the "order: false" line (which is mentioned in [cosmos theme readme](https://github.com/cosmos/vuepress-theme-cosmos#file-configuration)), it doesn't seem like it is needed when setting up translated folders, none of them are added to the English version navigation.

Reply:
👍